### PR TITLE
add rotate case

### PR DESCRIPTION
**Pull request by kychennv**
this case will be passed if bug 200528932 is  fixed
Signed-off-by: kychennv <kychen@nvidia.com>

### CODE REVIEW

**Comment by kychennv** [comment]
> I think you can add this tests to `test_operator_python_function.py` or rename this file to `rotate.py`.
   the name of the tests is followed by others test name: test_operator_flip.py, test_operator_slice.py,,
 also i have other tests about rotate rest,  and i will add them when the bug 200529317  fixed

**Comment by kychennv** [comment]
> I think you can add this tests to `test_operator_python_function.py` or rename this file to `rotate.py`.


the name of the tests is followed by others test name: test_operator_flip.py, test_operator_slice.py,,
also i have other tests about rotate rest, and i will add them when the bug 200529317 fixed

**Comment by JanuszL** [comment]
Yes, my mistake, I meant `test_operator_rotate.py`.

**Comment by kychennv** [comment]
> Yes, my mistake, I meant `test_operator_rotate.py`.

it is OK, i have modified now. sorry for that.

File: dali/test/python/test_operator_rotate.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        return images
+
+def check_rotate_cpu_vs_gpu(batch_size, angle):
+    compare_pipelines(RotatePipeline('cpu', batch_size, angle),
+                      RotatePipeline('gpu', batch_size, angle),
+                      batch_size=batch_size, N_iterations=10)
+
+def test_rotate_cpu_vs_gpu():
+    for batch_size in {1, 32, 100}:
+        for angle in range(0,360):
```
Comment (jantonguirao) [inline dali/test/python/test_operator_rotate.py]: how long does it take to run those tests? We are creating `1080` combinations here

Comment (kychennv) [reply dali/test/python/test_operator_rotate.py]: yes, does it need to reduce the tests?  such like set a step for range

Comment (kychennv) [reply dali/test/python/test_operator_rotate.py]: but the running time is not too long, i think maybe some minutes

Comment (jantonguirao) [reply dali/test/python/test_operator_rotate.py]: I think a step of ~20 degrees will be enough

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
Rotate tests were added in a different PR. Closing

**Pull request closed without merging**